framework/build.mk

```diff
@@ -29,7 +29,7 @@ libmesh_CC       ?= $(shell METHOD=$(METHOD) $(libmesh_config) --cc)
 libmesh_F77      ?= $(shell METHOD=$(METHOD) $(libmesh_config) --fc)
 libmesh_F90      ?= $(shell METHOD=$(METHOD) $(libmesh_config) --fc)
 libmesh_INCLUDE  := $(shell METHOD=$(METHOD) $(libmesh_config) --include)
-libmesh_CPPFLAGS := $(shell METHOD=$(METHOD) $(libmesh_config) --cppflags)
+libmesh_CPPFLAGS := $(shell METHOD=$(METHOD) $(libmesh_config) --cppflags) $(ADDITIONAL_CPPFLAGS)
```

Comment:
I think I'd rather keep the libMesh flags pristine. If you need a new variable add it to the rule itself.